todo-app/src/Task.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import TaskText from './TaskText';
+import TaskDate from './TaskDate';
+import './task.css';
+
+export default class Task extends React.Component {
+	
+  constructor(props) {
+    super(props);
+
+    // Events and bind to `this`
+    this.markAsDone = this.markAsDone.bind(this);
+  }
+
+  markAsDone(event) {
```

Comment:
This function should probably be renamed, as it does not _only_ mark things as done. It toggles their "done" state.